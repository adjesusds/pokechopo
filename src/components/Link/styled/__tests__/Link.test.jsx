// External Packages
import React from 'react'
import { create } from 'react-test-renderer'

// Internal Packages
import StyledLink from '../Link'

describe('<StyledLink />', () => {
  it('should style the component on valid props', () => {
    const styledComponent = create(<StyledLink />)

    expect(styledComponent).toMatchSnapshot()
  })
})
