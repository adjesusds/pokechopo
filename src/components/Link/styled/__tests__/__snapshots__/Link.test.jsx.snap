// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StyledLink /> should style the component on valid props 1`] = `
.c0 {
  color: #ffffff;
  text-decoration-skip: ink;
  -webkit-transition: color 200ms ease-in;
  transition: color 200ms ease-in;
}

.c0:hover {
  color: #ffeb3b;
}

<a
  className="c0"
/>
`;
